.pre-commit-config.yaml

```diff
@@ -1,17 +1,21 @@
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
+    rev: v4.4.0
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
     -   id: check-yaml
     -   id: check-added-large-files
 -   repo: https://github.com/pre-commit/mirrors-clang-format
-    rev: 'v14.0.0'
+    rev: v16.0.0
     hooks:
     -   id: clang-format
 -   repo: https://github.com/psf/black
-    rev: 22.10.0
+    rev: 23.1.0
     hooks:
     -   id: black
         args: [--config=python/pyproject.toml]
+-   repo: https://github.com/ahans/cmake-format-precommit
```

Comment:
Looked like a good chance to also include cmake-format, since you have a `.cmake-format.yaml`  in here (I didn't even know this exists, I'm not actively using CMake in any project). But it looks like neither the cmake-format project nor the pre-commit are very well maintained. cmake-format is missing some yaml dependency. The right solution would have been to fix that, but since then we'd also have to touch the hook, I forked the hook and put the dep there.

Reply:
I also tried and was not working 😂 thanks for providing a solution!